ApexOSA/osa.py

```
@@ -0,0 +1,1217 @@
+                    Max = [Peaks[0][Index], Peaks[1][Index]]
+                else:
+                    Max = max(Peaks)
+                return Max
+            else:
+                if Dual:
+                    return [0.0, 0.0]
+                else:
+                    return 0.0
+
+        self.__tracenumber = TraceNumber
+        return Peak
```

Ruff (0.12.2)

Undefined name `Peak`

(F821)